base/changes.txt

```diff
@@ -6,6 +6,11 @@ to completeness or accuracy and it contains some references to files that are
 not part of the distribution.
 ================================================================================
 
+2024-09-02  Joseph Wright  <Joseph.Wright@latex-project.org>
+
+	* clsguide.tex:
+	Document \IfPackageAtLeastTF, etc.
```

Comment:
Maybe also mention `\IfPackageLoadedTF`.
```suggestion
	Document \IfPackageLoadedTF, \IfPackageAtLeastTF, etc.
```

(I noticed it in the last round but didn't post it.)